src-ui/lib/components/BooksTable.svelte

```diff
@@ -1,8 +1,54 @@
 <script lang="ts">
+  import 'bulma/css/bulma.css'
+  import onScan from 'onscan.js'
   import { books } from '../state/Books.svelte'
+
+  import type { Action } from 'svelte/action'
+  import { getByISBN } from '../../lib/openLibrary.js'
+  import { type Book } from '../../lib/types/book.js'
   import BooksTableRow from './BooksTableRow.svelte'
+
+  const initialBook: Book = {
+    isbn10: '1234567890',
+    title: 'Hunting Prince Dracula',
+    tags: ['Young Adult', 'Fiction'],
+    authors: ['Kerri Maniscalco'],
+    hasRead: true,
+  }
+
+  if (books.value.length == 0) {
+    books.add(initialBook)
+  }
```

Comment:
In a future PR move this into the library that manages book state. Possibly with an `init` function called from the top level page itself.